<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{ 
	background-color: rgb(180, 180, 180);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="newButton">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Novo Contato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewButton">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>150</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Visualizar Contatos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editButton">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Editar Contato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton">
    <property name="geometry">
     <rect>
      <x>139</x>
      <y>270</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Remover Contato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(180, 180, 180);
	border-radius: 5px;
	padding: 2px;
	color: rgb(255, 255, 255);
}
QPushButton:hover{
	background-color: rgb(60, 60, 60);
	border: 2px solid rgb(55, 55, 55);
}
QPushButton:pressed{
	background-color: rgb(250, 230, 0);
	border: 2px solid rgb(255, 165, 24);
}</string>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>129</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Agenda de Contatos:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
